modules/restore-add-master.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-add-master_{context}"]
+= Adding a master host back to the etcd cluster
+
+Follow these steps to add a master host back to the etcd cluster. This procedure assumes that you previously removed the master host from the cluster and that its etcd dependencies are valid, such as TLS certificates and DNS.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* SSH access to the master host to add to the etcd cluster.
+* The IP address of an existing active etcd member.
+
+.Procedure
+
+. Access the master host to add to the etcd cluster.
++
+[IMPORTANT]
+====
+This procedure _must_ be run on the master host being added to the etcd cluster.
```

Comment:
Because it is already called out in an important note:
s/_must_/must/

Also, "must be run" sounds odd to me. Maybe try "must run", "must be executed", or, "You must run this procedure on the master host that is being added to the etcd cluster."